src/status_im/contexts/keycard/login/events.cljs

```diff
@@ -0,0 +1,71 @@
+(ns status-im.contexts.keycard.login.events
+  (:require [re-frame.core :as rf]
+            [status-im.contexts.keycard.utils :as keycard.utils]
+            [taoensso.timbre :as log]
+            [utils.transforms :as transforms]))
+
+(rf/reg-event-fx :keycard.login/on-get-keys-error
+ (fn [{:keys [db]} [error]]
+   (log/debug "[keycard] get keys error: " error)
+   (let [tag-was-lost?     (keycard.utils/tag-lost? (:error error))
+         pin-retries-count (keycard.utils/pin-retries (:error error))]
+     (if tag-was-lost?
+       {:db (assoc-in db [:keycard :pin :status] nil)}
+       (if (nil? pin-retries-count)
+         {:effects.utils/show-popup {:title "wrong-keycard"}}
+         {:db (-> db
+                  (assoc-in [:keycard :application-info :pin-retry-counter] pin-retries-count)
+                  (update-in [:keycard :pin] assoc :status :error))
+          :fx [[:dispatch [:keycard/hide-connection-sheet]]
+               (when (zero? pin-retries-count)
+                 [:effects.utils/show-popup {:title "frozen-keycard"}])]})))))
+
+(rf/reg-event-fx :keycard.login/on-get-keys-success
+ (fn [{:keys [db]} [data]]
+   (let [{:keys [key-uid encryption-public-key
+                 whisper-private-key]} (transforms/js->clj data)
+         key-uid                       (str "0x" key-uid)
+         profile                       (get-in db [:profile/profiles-overview key-uid])]
```

Comment:
it's not good to use `name` symbol, because its in cljs core, and I don't like something like `profile-name`

Reply:
I agree with you about `name` shadowing a core Clojure function. In `clj-kondo/config.edn` I left only two exceptions which are too common in `status-mobile`:

```clojure
:shadowed-var {:level   :error
               ;; We temporarily use :include to define an
               ;; allowlist of core Clojure vars. In the
               ;; future, as we progressively fix shadowed
               ;; vars, we should be able to delete this
               ;; option and lint all vars.
               :exclude [type name]}
```

To be realistic, I think we'll never remove `name` and `type` from the `:exclude` rule. There are hundreds of cases and it makes destructuring in `status-mobile` quite inconvenient. So that's why I suggested because it's an allowed "exception" in our linter.